### ISSUE TITLE
Fix span display

### ISSUE THREAD

**Issue by romanhabibov** [tarantool/tarantool#4786]
From #3962

```
tarantool> box.space._session_settings:update({'sql_full_metadata'}, {{'=', 'value', true}})
---
- ['sql_full_metadata', true]
...
tarantool> box.execute('VALUES (1, 2, 3)')
---
- metadata:
  - type: integer
    span: column1  !! should be '1'
    name: column1
  - type: integer
    span: column2  !! should be '2'
    name: column2
  - type: integer
    span: column3  !! should be '3'
    name: column3
  rows:
  - [1, 2, 3]
...
```

**Comment by Korablev77** [comment]
Why don't you fix it in scope of #3962?

**Comment by romanhabibov** [comment]
> Why don't you fix it in scope of #3962?

I thought, maybe this is not so important in the current milestone. And I did not find a quick fix at first glance.

**Comment by Korablev77** [comment]
@romanhabibov Let's deal with all naming related at once in scope of current release. Otherwise there's still chance that name/span/default will be in inconsistent state ergo require second fixing iteration. Please elaborate on that.
